.github/workflows/iceberg.yml

```diff
@@ -0,0 +1,68 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
```

Comment:
Borrowed from https://github.com/apache/auron/issues/1541 as its not merged in yet.